lib/bibdata_rs/src/theses/holdings.rs

```diff
@@ -81,19 +83,19 @@ pub fn on_site_only(
     location: bool,
     access_rights: bool,
     mudd_walkin: bool,
-    class_year: &[String],
+    class_year: Vec<String>,
```

Comment:
Is there a way we can keep the old signature?  It would be more flexible to not require an owned `Vec`